imap_processing/codice/codice_l1a.py

```diff
@@ -289,13 +330,19 @@ def get_params(packets):
     view_id : int
         Provides information about how data was collapsed and/or compressed
     """
-    # TODO: Once all APIDs are supported, these numbers could be randomized for
-    # testing purposes
-
+    # table_id will likely always be 1
     table_id = 1
-    plan_id = 1
-    plan_step = 1
-    view_id = 5
+
+    if apid == CODICEAPID.COD_LO_SW_SPECIES_COUNTS:
```

Comment:
Is the `view_id` the only one that is changing?

Reply:
Hard to say. This is one of those things on CoDICE where certain knobs were built to allow a lot of configuration, but in practice/reality it could be that the `plan_id` and `plan_step` don't change much, or at all. I think it is still up in the air.